pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -58,20 +58,18 @@ def load_earth_magnetic_anomaly(
         ``data_source="emag2_4km"``, which are ``"pixel"`` only.
 
     data_source : str
-        Select the source of the magnetic anomaly data.
+        Select the source of the magnetic anomaly data. Available options are:
 
-        Available options:
-
-        - **emag2** : EMAG2 Global Earth Magnetic Anomaly Model [Default
+        - ``"emag2"`` : EMAG2 Global Earth Magnetic Anomaly Model [Default
```

Comment:
Is there an intention behind setting a white space before the `:`?
```suggestion
        - ``"emag2"``: EMAG2 Global Earth Magnetic Anomaly Model [Default
```